src/test/java/com/karankumar/bookproject/backend/utils/CustomShelfUtilsTest.java

```diff
@@ -82,20 +84,20 @@ private static void saveCustomShelves(CustomShelf... customShelves) {
     }
 
     @Test
-    void getBooksInCustomShelfSuccessfullyReturnsBooks() {
+    void returnBooksSuccessfully() {
         Set<Book> actual = customShelfUtils.getBooksInCustomShelf(customShelf1.getShelfName());
         booksInCustomShelf1.forEach(book -> assertThat(actual).contains(book));
     }
 
     @Test
-    void givenNoBooksInCustomShelf_getBooksInCustomShelfReturnsNoBooks() {
+    void returnNoBooksIfNoBooksInCustomShelf() {
         Set<Book> actual =
                 customShelfUtils.getBooksInCustomShelf(customShelfWithNoBooks.getShelfName());
         assertThat(actual).isEmpty();
     }
 
     @Test
-    void givenInvalidCustomShelf_anEmptySetOfBooksAreReturned() {
+    void returnEmptySetOfBooksWhenCustomShelfIsInvalid() {
```

Comment:
Could you please rename this to 'returnEmptySetofBooksFromNonExistentCustomShelf'?